lutris/config.py

```diff
@@ -36,6 +36,18 @@ def duplicate_game_config(game_slug: str, source_config_id: str):
     return new_config_id
 
 
+def rename_config(old_config_id: str, new_slug: str):
```

Comment:
I think you should specify the return type here.

~~But perhaps it would be better if you did not return ``None``, but instead ``old_config_id``, so the return type can be ``str``- and the one place that calls this can unconditionally assigned it, without bringing a walrus into it.~~ Never mind, this won't work- you still want the new config id, not the old, and you do not have it here. ``-> Optional[str]`` is correct here.